soh/soh/Enhancements/randomizer/3drando/item_pool.cpp

```diff
@@ -662,9 +662,17 @@ void GenerateItemPool() {
     IceTrapModels.push_back(0xD3);
   }
 
+  if (TriforceHunt.Is(TRIFORCE_HUNT_ON)) {
+    IceTrapModels.push_back(0xDF);
+    AddItemToMainPool(TRIFORCE_PIECE, Settings::TriforceHuntTotal.Value<uint8_t>());
+    PlaceItemInLocation(TRIFORCE_COMPLETED, TRIFORCE); // Win condition
+    PlaceItemInLocation(GANON, GetJunkItem(), false, true);
```

Comment:
not a v1 thing, but it could be cool to have a setting so beating ganon could have a non-junk item

Reply:
I had a small bit of discussion with @Archez about multiple "goal" settings as well, which would basically mean you could combine triforce hunt AND defeating ganon as the goal. As in, triforce hunt would grant the boss key which in turn would lead to being able to kill ganon.

There's a bit of extra work required to do this though, but it could be really cool also allowing the reverse yeah, where Ganon is potentially a check and triforce hunt remains the final goal.